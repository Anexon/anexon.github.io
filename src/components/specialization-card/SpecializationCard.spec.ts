import SpecialiationCard from './SpecializationCard';
import TestRenderer from 'react-test-renderer';

it('Renders correctly', () => {
    const card = TestRenderer.create(SpecialiationCard({
        imgUrl: "una url",
        title: "Título",
        description: "Descripción",
        isHighlighted: false
    })).toJSON();
    expect(card).toMatchSnapshot();
})