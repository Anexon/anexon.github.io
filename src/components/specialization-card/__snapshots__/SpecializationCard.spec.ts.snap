// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="specialization-card-container show-on-scroll "
>
  <img
    src="una url.svg"
  />
  <h3>
    Título
  </h3>
  <p>
    Descripción
  </p>
</div>
`;
